ocaml/xs-trace/dune

```diff
@@ -19,8 +19,9 @@
   (action (with-stdout-to %{targets} (run %{exe} --help=groff)))
 )
 
-(install
-  (section man)
-  (package xapi)
-  (files (xs-trace.1 as man1/xs-trace.1))
-)
+; not expected by the specfile
+;(install
```

Comment:
While the specfile doesn't some manpages, I would like to retain them

Reply:
We can add them back in a followup PR that also modifies the specfile.
(I want to separate the changes that do not require specfile changes from the ones that can, because that way I can verify that most changes do not result in changes to the list of installed files, whereas with specfile changes we do expect changes to the list of installed files).

I have started to collect those changes here: https://github.com/edwintorok/xen-api/compare/private/edvint/packageopt0...edwintorok:xen-api:private/edvint/packageopt1